packages/dds/tree/src/test/shared-tree/fuzz/composeVsIndividual.fuzz.spec.ts

```diff
@@ -130,6 +130,7 @@ describe("Fuzz - composed vs individual changes", () => {
 		createDDSFuzzSuite(model, {
 			defaultTestCount: runsPerBatch,
 			numberOfClients: 1,
+			detachedStartOptions: { numOpsBeforeAttach: 0 },
```

Comment:
is this how you are disabling rehydrate? doesn't this also disable attach without rehydrate? if so, can we add a way to only disable rehydrate and not attach